tests/umock_c_negt_int/umock_c_negt_int_tests.c

```diff
@@ -301,7 +301,7 @@ TEST_FUNCTION(negative_tests_with_1_call_with_void_return_dependency)
 TEST_FUNCTION(negative_tests_with_1_call_with_void_ptr_return_dependency)
 {
     size_t i;
-    STRICT_EXPECTED_CALL(function_3_void_ptr_return(IGNORED_PTR_ARG))
+    STRICT_EXPECTED_CALL(function_3_void_ptr_return(IGNORED_ARG))
         .SetReturn((void*)0x42).SetFailReturn(NULL).IgnoreArgument_a();
```

Comment:
>IgnoreArgument_a [](start = 52, length = 16)

Isn't it already ignored at this point? Because of IGNORED_ARG?

Reply:
It is. Removed the explicit ignore with .IgnoreArgument_a();

---
In reply to: [395826770](https://github.com/Azure/umock-c/pull/152#discussion_r395826770) [](ancestors = 395826770)